services/editor/src/pages/keys/components/KeysList/KeysList.js

```diff
@@ -36,7 +37,12 @@ const KeyItem = connect((state, props) => ({
   </div>,
 );
 
+const withoutInternal = list => (list ? list.filter(x => !/^@tweek\//.test(x)) : list);
+
 const KeysList = componentFromStream((prop$) => {
+  const showInternal$ = Observable.fromPromise(
+    fetch('/api/editor-configuration/show_internal_keys').then(response => response.json()),
+  );
   const keyList$ = prop$.map(x => x.keys).distinctUntilChanged();
```

Comment:
http://reactivex.io/rxjs/class/es6/Observable.js~Observable.html#instance-method-withLatestFrom
filter this observable